brokenspoke_analyzer/core/ingestor.py

```diff
@@ -183,6 +179,11 @@ def import_neighborhood(
         load_water_blocks(engine, water_blocks_file)
         delete_water_blocks(engine)
 
+    transform_query = (
+        f"ALTER TABLE {CENSUS_BLOCKS_TABLE} RENAME COLUMN geoid20 TO blockid20"
+    )
+    dbcore.execute_query(engine, transform_query)
```

Comment:
As mentioned previously, this will likely not be needed.

Reply:
Sounds good! Deleted this and switched the `blockid20`s to `geoid20`s for this table. I wasn't sure how to handle the other tables, so I've left them as `blockid20`.